spec/unit/type/file/content_spec.rb

```diff
@@ -205,6 +205,12 @@
               expect(content).not_to be_safe_insync("other content")
             end
 
+            it "prints binary file notice if diff is not valid encoding" do
+              expect(content).to receive(:diff).and_return("\xc7\xd1\xfc\x84").once
+              expect(content).to receive(:debug).with(/\nBinary files #{filename} and .* differ/).once
```

Comment:
nit, I usually avoid saying `.once`, as it's redundant. From the [rspec docs](https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/receive-counts)

> If you don't specify an expected receive count, it defaults to once.

But it looks like other tests have copy and pasted that behavior since it was introduced in https://github.com/puppetlabs/puppet/commit/136949da70f1a6715734faa6ac3c464f386fccfb. Feel free to ignore my niit or add a `(maint)` commit to remove all of the `.once` in this file.